generated/py/proto/delivery/delivery_pb2.pyi

```diff
@@ -324,9 +324,9 @@ class Insertion(google.protobuf.message.Message):
     """Optional.  We'll look this up using the external_content_id."""
 
     position: builtins.int = ...
-    """Optional. 0-based. Position "in what" depends on insertion context:
-    if request_insertion, then position provided by client or retrieval
-    if response_insertion, then the position returned by Delivery to the client
+    """Optional. 0-based.  Position is the global position across pages.
+    E.g. the first item on the 4th page where page_size=10 is position=40.
+    Do not set this on request insertions.  Instead, set retrieval_rank.
```

Comment:
Who is this advice for? Is it for me when I'm reading the request insertion and setting fields or for metrics when actually writing out the file? I'm trying to figure out if I need to make additional changes to follow this guidance or not.

Reply:
This is external docs and in the external repo.